pom.xml

```diff
@@ -61,34 +61,39 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<java.version>1.8</java.version>
 		
-		<sonar.projectKey>revaturelabs_rideshare-user-service</sonar.projectKey>
-  		<sonar.organization>revaturelabs-screenforce</sonar.organization>
+		<sonar.projectKey>Batch-738-Mathew-Oberlies_RideshareBackend</sonar.projectKey>
+  		<sonar.organization>batch-738-mathew-oberlies</sonar.organization>
   		<sonar.host.url>https://sonarcloud.io</sonar.host.url>
-  		<sonar.coverage.exclusion>com/revature/beans/*.java</sonar.coverage.exclusion>
+  		<sonar.login>960c465fd2b35a880eb21c0323f22ffac996be6b</sonar.login>
```

Comment:
It might be better to move this out of the pom.xml. We can instead modify the pipeline to include the relevant credentials using a flag in the mvn sonar:sonar command. That way we can also store credentials as a environment variable. Perhaps it would be prudent to leave a comment explaining that sonarcloud is configured in the pipeline. It might even be nice to add a template example to the readme? Although those are of course optional.